apps/css-workshop/src/components/Icon.astro

```diff
@@ -0,0 +1,22 @@
+---
+type Props = {
+  svg: string;
+  size?: 's' | 'm' | 'l' | 'auto';
+  padded?: boolean;
+  fill?: string;
+} & astroHTML.JSX.HTMLAttributes;
+
+const { svg, size = 'm', class: className, padded, fill = 'default', ...rest } = Astro.props;
+
+const iconProps = {
```

Comment:
This intermediate variable is not necessary. The props can be used directly in the markup.

```jsx
<Element
  class:list={['iui-svg-icon', className]}
  …
```

Reply:
Nice, much cleaner, removed.

Reply:
You've only moved one (1) prop in total. The `…` in my suggestion is meant to imply that you can remove the whole `iconProps` object 😅

Reply:
Ah, I misunderstood.  All `iconProps` have been inlined now.